src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -350,10 +354,11 @@ public void canOverrideRenewalWhenDateFallsOutsideOfTheDateRangesInTheFixedLoanP
     final IndividualResource loanPolicy = loanPoliciesFixture.create(currentDueDateRollingPolicy);
     UUID dueDateLimitedPolicyId = loanPolicy.getId();
 
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
Can you replace this by `use(loanPolicy)`?

And several similar `useFallbackPolicies()` invocations in this file?

Reply:
Done.